documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -2098,7 +2279,7 @@ include::{partialsdir}/modules/all-connectors/ref-connector-monitoring-streaming
 // ModuleID: debezium-sql-server-connector-schema-history-metrics
 // Title: {prodname} SQL Server connector schema history metrics
 [[sqlserver-schema-history-metrics]]
-==== Schema history metrics
+==== Schema History Metrics
```

Comment:
I think the title casing still is rather inconsistent. I don't think we used upper-casing most of the times. So perhaps refrain from doing it here, _or_ do it consistently everywhere?